pkgs/development/python-modules/django-mdeditor/Bump-KaTeX-and-replace-bootcdn-with-jsdelivr.patch

```diff
@@ -0,0 +1,63 @@
+From c5af641cccf663dffb4a47d32e28404f609badce Mon Sep 17 00:00:00 2001
+From: Tomo <tomodachi94@protonmail.com>
+Date: Sat, 12 Oct 2024 03:39:12 +0000
+Subject: [PATCH 1/2] chore(KaTeX): bump to 0.7.1
+
+Many bugfixes. This KaTeX is still quite old,
+but versions beyond this have backwards-incompatibilities
+(starting in 0.8).
+---
+ mdeditor/static/mdeditor/js/editormd.js | 4 ++--
+ 1 file changed, 2 insertions(+), 2 deletions(-)
+
+diff --git a/mdeditor/static/mdeditor/js/editormd.js b/mdeditor/static/mdeditor/js/editormd.js
+index be0005d..8aacb56 100644
+--- a/mdeditor/static/mdeditor/js/editormd.js
++++ b/mdeditor/static/mdeditor/js/editormd.js
+@@ -4179,8 +4179,8 @@
+     // 使用国外的CDN，加载速度有时会很慢，或者自定义URL
+     // You can custom KaTeX load url.
+     editormd.katexURL  = {
+-        css : "//cdn.bootcdn.net/ajax/libs/KaTeX/0.3.0/katex.min",
+-        js  : "//cdn.bootcdn.net/ajax/libs/KaTeX/0.3.0/katex.min"
++        css : "//cdn.bootcdn.net/ajax/libs/KaTeX/0.7.1/katex.min",
```

Comment:
I'm guessing you to depend on cdn.jsdelivr.net here instead :)

Reply:
Nope. I split the version bump and the URL change into two separate commits. There are two patches in this patch.

Reply:
See further down in the file: https://github.com/tomodachi94/nixpkgs/blob/ced695af7883191ea40a96b5304e5794d3ce8110/pkgs/development/python-modules/django-mdeditor/Bump-KaTeX-and-replace-bootcdn-with-jsdelivr.patch#L35

Reply:
Yup sorry, I just misread the patch :)